koku/masu/processor/parquet/parquet_report_processor.py

```diff
@@ -487,6 +487,10 @@ def convert_to_parquet(self):  # noqa: C901
                     )
                 )
                 raise ParquetReportProcessorError(msg)
+        if self.create_table and not self.trino_table_exists.get(self.report_type):
+            self.create_parquet_table(parquet_base_filename)
+        if self.provider_type not in (Provider.PROVIDER_AZURE):
+            self.create_parquet_table(parquet_base_filename, daily=True)
```

Comment:
What changed to make this addition now necessary?

Reply:
This is regarding the sync partitions piece we discussed. `create_parquet_table` is what triggers the sync partitions and we wanted to run this after all processed files, this was an attempt to do that outside the for loop.

Reply:
Is moving this code related to the invoice_month piece? Maybe that should be done in a different PR.

Reply:
Yes its related, the partitions wont get synced correctly otherwise. :disappointed:

Reply:
I guess we could do it as a pre-req if we wanted, but right now it blocks this at least.

Reply:
Right it was not fully tested, I just threw this together as a start for Lucas. :wink: 

As for the partition sync piece, this was what we discussed on the triage the other week. If you process GCP files cross month bounds we only sync the first months partition rather then once for both months. That lead to us missing the second partition since it wasn't synced. The net of that discussion was to move it outside the loop so we only sync once at the end. It's possible I just missed another change when I pushed this up though because I definitely tested GCP and thought I fixed that exception. :disappointed: